Source/WebCore/dom/DocumentMarkerController.cpp

```diff
@@ -785,7 +785,7 @@ void DocumentMarkerController::showMarkers() const
     for (auto& nodeMarkers : m_markers) {
         fprintf(stderr, "%p", nodeMarkers.key.ptr());
         for (auto& marker : *nodeMarkers.value)
-            fprintf(stderr, " %hu:[%d:%d]", marker.type(), marker.startOffset(), marker.endOffset());
+            fprintf(stderr, " %hu:[%d:%d]", static_cast<uint16_t>(marker.type()), marker.startOffset(), marker.endOffset());
```

Comment:
`enumToUnderlyingType(marker.type())` would have been a little nicer I think.

Reply:
Thank you. Will fix in another PR.